Docs/developer_guide/build_instructions/windows.md

```diff
@@ -56,20 +56,30 @@ Specify source, build, and Qt location and compiler version and start the build
 
 Release mode:
 
-```
+```bat
+cd C:\D
 mkdir C:\D\S4R
-cd /d C:\D\S4R
-"C:\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 17 2022" -A x64 -DQt5_DIR:PATH=C:\Qt\5.15.2\msvc2019_64\lib\cmake\Qt5 C:\D\S4
-"C:\Program Files\CMake\bin\cmake.exe" --build . --config Release
+
+cmake ^
```

Comment:
@benzwick For consistency, the instructions in `developer_guide/extensions.md` should also be updated.


@pieper @lassoan @jamesobutler

I suggested simply using `cmake` instead of the more verbose `"C:\Program Files\CMake\bin\cmake.exe"`

Reply:
The default option for the cmake installer (at least on Windows) is to not add CMake to the system PATH. A novice builder may run into issues here

![image](https://github.com/Slicer/Slicer/assets/15837524/65ac3002-55f0-478e-b9ae-dd5afcae9116)

Reply:
I agree, I would vote for keeping `C:\Program Files\CMake\bin\cmake.exe` because it works by default, while `cmake` does not.

Reply:
OK, I guess I forgot that I selected that option when I installed CMake.

Reply:
I changed it back to `C:\Program Files\CMake\bin\cmake.exe`